app/indexers/administrative_tag_indexer.rb

```diff
@@ -24,10 +24,15 @@ def to_solr
       solr_doc['exploded_tag_ssim'] += exploded_tags_from(tag)
 
       tag_prefix, rest = tag.split(TAG_PART_DELIMITER, 2)
-      next if TAGS_TO_INDEX.exclude?(tag_prefix) || rest.nil?
+      next if SPECIAL_TAG_TYPES_TO_INDEX.exclude?(tag_prefix) || rest.nil?
 
       prefix = tag_prefix.downcase.strip.gsub(/\s/, '_')
+
       (solr_doc["#{prefix}_tag_ssim"] ||= []) << rest.strip
+      if prefix == 'project'
+        solr_doc['exploded_project_tag_ssim'] ||= []
+        solr_doc['exploded_project_tag_ssim'] += exploded_tags_from(rest.strip)
```

Comment:
these lines do the other part of what the ticket asked for (i got a syntax error when i tried to do them as one line, i.e. `(solr_doc['exploded_project_tag_ssim'] ||= []) += exploded_tags_from(rest.strip)`)

Reply:
also, i'd originally added indexing for exploded versions of both the project tag and the registered by tag.  but from a quick glance at the argo prod data, it didn't seem like registered by tags ever had multiple levels.  so an exploded version didn't seem particularly useful, and the ticket didn't ask for it, so i kept this change confined to project tags.